lib/StencilConfigManager.js

```diff
@@ -99,14 +99,22 @@ class StencilConfigManager {
      * @param {boolean} ignoreMissingFields
      * @returns {object}
      */
-    _validateStencilConfig(config, ignoreMissingFields) {
+    _validateStencilConfig(configFile, ignoreMissingFields) {
+        const config = configFile;
+
         if (!ignoreMissingFields && (!config.normalStoreUrl || !config.customLayouts)) {
             throw new Error(
                 'Error: Your stencil config is outdated. Please run'.red +
                     ' $ stencil init'.cyan +
                     ' again.'.red,
             );
         }
+
+        if (!config.apiHost) {
+            this._logger.log(`No api host found in config file, falling back to ${API_HOST}`);
```

Comment:
Maybe add "You may need to run stencil init again" so the user knows how to fix this warning? 🍹